static/js/controllers/inbox.js

```diff
@@ -650,6 +650,12 @@ require('moment/locales');
         if (deleteMessageId) {
           DeleteDoc(deleteMessageId, function(err) {
             pane.done(translateFilter('Error deleting document'), err);
+            if (err) {
+              pane.done(translateFilter('Error deleting document'), err);
+              return;
+            }
+            pane.done();
+            $state.go('contacts');
```

Comment:
Typo alert - three calls to `pane.done()` here!  My preference would be to keep the first one.